bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/gnirs/GnirsRule.java

```diff
@@ -43,6 +43,33 @@ public IConfigMatcher getMatcher() {
         }
     };
 
+    private static final IConfigRule SHORT_RED_CAMERA_RULE = new IConfigRule() {
+        private static final String MESSAGE = "The short red camera is currently unavailable.";
+
+        @Override
+        public Problem check(final Config config, final int step, final ObservationElements elems, final Object state) {
+            final Camera camera = (Camera) SequenceRule.getInstrumentItem(config, InstGNIRS.CAMERA_PROP);
+            if (camera == null) return null;
+            switch (camera) {
+                case SHORT_BLUE:
+                case LONG_BLUE:
+                    return null;
+            }
+
+            final PixelScale ps = (PixelScale) SequenceRule.getInstrumentItem(config, InstGNIRS.PIXEL_SCALE_PROP);
+            if (ps != PixelScale.PS_015) return null;
+            final Wavelength l = (Wavelength) SequenceRule.getInstrumentItem(config, InstGNIRS.CENTRAL_WAVELENGTH_PROP);
+            if ((l == null) || l.doubleValue() <= 2.5) return null;
```

Comment:
I'm missing something.  It seems like the `SHORT_RED` camera is available or it isn't.   If the pixel scale is `PS_015` or the central wavelength less than 2.5 does it become available?

Reply:
My understanding is that there is no SHORt_RED component aa such but it is implicit when the pixel scale and central wavelength combine in a set of values

Reply:
That may be so but in the OT at least, you can explicitly choose `SHORT_RED` as an option in the sequence component regardless of the pixel scale and wavelength.